server/chat-plugins/room-faqs.ts

```diff
@@ -120,6 +120,10 @@ export const commands: ChatCommands = {
 
 		if (!this.runBroadcast()) return;
 		this.sendReplyBox(Chat.formatText(roomFaqs[room.roomid][topic], true));
+		if (!this.broadcasting && user.can('ban', null, room, 'rfaq')) {
+			const arg = roomFaqs[room.roomid][topic];
+			this.sendReplyBox(Chat.getReadmoreCodeBlock(`/addfaq ${topic}, ${arg}`));
+		}
```

Comment:
I think sources should be entirely hidden by default.

```suggestion
		if (!this.broadcasting && user.can('ban', null, room, 'rfaq')) {
			const code = Utils.escapeHTML(roomFaqs[room.roomid][topic]).replace(/\n/g, '<br />');
			this.sendReplyBox(`<details><summary>Source</summary><code style="white-space: pre-wrap; display: table; tab-size: 3">/addfaq ${topic}, ${arg}</code></details>`);
		}
```

This matches `/roomintro` better, too.

Reply:
Indeed, it should also be done for `/roomevent view `?

Reply:
Possibly. Do you want to do that in this PR, too?

Reply:
Yes i will do that

Reply:
I can't seem to do it in this PR. Can I do another one?